src/third-party/gutenberg.stories.mdx

```diff
@@ -0,0 +1,568 @@
+            <strong>Snow Patrol</strong>
+          </p>
+        </div>
+      </div>`}
+  </Story>
+</Preview>
+
+### Cover - Video Variant
+
+<Preview>
+  <Story name="core/cover/video" height="550px">
+    {`<div class="wp-block-cover has-background-dim">
+        <div class="vsc-controller" data-vscid="g0pcyj7av">
+        </div>
+        <video
+          class="wp-block-cover__video-background"
+          autoplay
+          muted
+          src="http://mirrors.standaloneinstaller.com/video-sample/dolbycanyon.m4v"
+          data-origwidth="0"
+          data-origheight="0"
+          style="width: 580px;"
+          data-vscid="g0pcyj7av">
+        </video>
+        <div class="wp-block-cover__inner-container">
+          <p class="has-text-align-center has-large-font-size">Video</p>
+        </div>
+      </div>`}
+  </Story>
+</Preview>
+
+## Embed
+
+The embed block wraps any embedded content in a `figure` element with class
+`wp-block-embed` and an inner `div` with class `wp-block-embed__wrapper`. Many
+specific embed block types build on this core embed block, wrapping a specific embedded
+content and adding `is-type-embedType-embed`, `wp-block-embed-embedType` and
+`is-provider-providerType` classes to the parent wrapper element.
+
+Adding a caption in the editor includes a `figcaption` element before the closing
+`figure` tag.
+
+<Preview>
+  <Story name="core/embed" height="300px">
+    {`<figure
+        class="wp-block-embed-wordpress wp-block-embed is-type-wp-embed is-provider-wp-themes-com"
+      >
+        <div class="wp-block-embed__wrapper">
+          <blockquote
+            class="wp-embedded-content"
+            data-secret="AbCd1234"
+            style="display: none;"
+          >
+            <a href="https://wp-themes.com/twentytwenty/?p=36"
+              >Elements</a
+            >
+          </blockquote>
+          <iframe
+            title="Elements — wp-themes.com"
+            class="wp-embedded-content"
+            sandbox="allow-scripts"
+            security="restricted"
+            src="https://wp-themes.com/twentytwenty/?p=36/embed/#?secret=AbCd1234"
+            data-secret="AbCd1234"
+            width="580"
+            height="257"
+            frameborder="0"
+            marginwidth="0"
+            marginheight="0"
+            scrolling="no"
+            data-origwidth="580"
+            data-origheight="327"
+          ></iframe>
+        </div>
+      </figure>`}
+  </Story>
+</Preview>
+
+## File
+
+The file block allows downloading or opening media or page attachment content.
+
+This block creates a `div` with class `wp-block-file`, which wraps one or two
+`a` elements, the second of which (toggled in the editor) is meant to be styled
+as a button, adding the class `wp-block-file__button`.
+
+<Preview>
+  <Story name="core/file" height="100px">
+    {`<div class="wp-block-file">
+        <a href="https://i1.wp.com/wordpress.org/news/files/2020/03/Squares.jpg"
```

Comment:
Let's avoid hotlinking to images in favor of storing some examples in our project. There should be plenty if you browse around our articles.

Reply:
For some reason, I avoided uploading assets into the repo (wasn't sure that was OK), but thought about the drawbacks of hotlinking. I will look to add images, audio, and video, as needed, because I think they're needed.

Reply:
I've localized all previously hotlinked assets under `static/media`